core-libs/setup/ssr/optimized-engine/rendering-cache.ts

```diff
@@ -0,0 +1,53 @@
+import { SsrOptimizationOptions } from './ssr-optimization-options';
+
+export interface RenderingEntry {
+  html?: any;
+  err?: any;
+  time?: number;
+  rendering?: boolean;
+}
+
+export class RenderingCache {
+  protected renderedUrls: {
+    [filePath: string]: RenderingEntry;
```

Comment:
```suggestion
    [requestKey: string]: RenderingEntry;
```

filePath seemed not appropiate for me here

Reply:
👍